admin-manual/installation/linux/ubuntu-bionic.rst

```diff
@@ -143,136 +143,6 @@ Start the service and configure it to start when the system is booted.
-   update the configuration snippets that we share later in this document
-   according to your location.
-
-.. _linux-ubuntu-bionic-dependency-httpd-nginx:
-
-Nginx
-`````
-
-In Ubuntu, the installation of Nginx is simple:
-
-.. code-block:: bash
-
-   sudo apt install nginx
-
-Nginx deploys a default server (aka VirtualHost, for Apache users) called
-**default** and you can find it in :file:`/etc/nginx/sites-available/default`.
-In order to install AtoM you could edit the existing server block or add a new
-one. We are going to you show you how to do the latter:
-
-.. code-block:: bash
-
-   sudo touch /etc/nginx/sites-available/atom
-   sudo ln -sf /etc/nginx/sites-available/atom /etc/nginx/sites-enabled/atom
-   sudo rm /etc/nginx/sites-enabled/default
-
-We have now created the configuration file and linked it from sites-enabled/,
-which is the directory that Nginx will look for. This means that you could
-disable a site by removing its symlink from sites-enabled/ while keeping the
-original one under sites-available/, in case that you want to re-use it in the
-future. You can do this with the Nginx default server.
-
-The following is a recommended server block for AtoM. Put the following contents
-in :file:`/etc/nginx/sites-available/atom`.
-
-.. code-block:: nginx
-
-   upstream atom {
-     server unix:/run/php7.2-fpm.atom.sock;
-   }
-
-   server {
-
-     listen 80;
-     root /usr/share/nginx/atom;
-
-     # http://wiki.nginx.org/HttpCoreModule#server_name
-     # _ means catch any, but it's better if you replace this with your server
-     # name, e.g. archives.foobar.com
-     server_name _;
-
-     client_max_body_size 72M;
-
-     # http://wiki.nginx.org/HttpCoreModule#try_files
-     location / {
-       try_files $uri /index.php?$args;
-     }
-
-     location ~ /\. {
-       deny all;
-       return 404;
-     }
-
-     location ~* (\.yml|\.ini|\.tmpl)$ {
-       deny all;
-       return 404;
-     }
-
-     location ~* /(?:uploads|files)/.*\.php$ {
-       deny all;
-       return 404;
-     }
-
-     location ~* /uploads/r/(.*)/conf/ {
-
-     }
-
-     location ~* ^/uploads/r/(.*)$ {
-       include /etc/nginx/fastcgi_params;
-       set $index /index.php;
-       fastcgi_param SCRIPT_FILENAME $document_root$index;
-       fastcgi_param SCRIPT_NAME $index;
-       fastcgi_pass atom;
-     }
-
-     location ~ ^/private/(.*)$ {
-       internal;
-       alias /usr/share/nginx/atom/$1;
-     }
-
-     location ~ ^/(index|qubit_dev)\.php(/|$) {
-       include /etc/nginx/fastcgi_params;
-       fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
-       fastcgi_split_path_info ^(.+\.php)(/.*)$;
-       fastcgi_pass atom;
-     }
-
-     location ~* \.php$ {
-       deny all;
-       return 404;
-     }
-
-   }
-
-Now you need to enable and reload Nginx:
-
-.. code-block:: bash
-
-   sudo systemctl enable nginx
-   sudo systemctl reload nginx
-
 .. _linux-ubuntu-bionic-dependency-php:
```

Comment:
Hi! Given that for the foreseeable future, we are just going to be maintaining one set of linux instructions, rather than changing these links to "focal, " I suggest we just say `linux-ubuntu-dependency-php` etc. If in the future we need to create a second Ubuntu LTS installation page, we can consider how to differentiate them at that time. 

That said, if you intend to address the PHP 7.4 and Focal upgrades in a different PR, feel free to ignore for now.

Reply:
I'll do it in this PR. Maybe just `ubuntu-dependency-php` then.